ml-proto/host/import/env.ml

```diff
@@ -0,0 +1,39 @@
+open Values
+open Types
+
```

Comment:
The name `env` is confusing to me. The name `waterfalltest` was suggested in this PR; I'd be ok with that.